tests/nnstreamer_filter_extensions_common/meson.build

```diff
@@ -8,53 +8,129 @@ tizen_apptest_deps = [
 ]
 
 # Format for adding subplugin into extensions -
-# [name, extension abbreviation, dependencies, model file name/folder path/file path, test name]
-extensions = []
-
-extensions += [['custom', 'custom', nnstreamer_unittest_deps, '../build/nnstreamer_example/custom_example_passthrough/libnnstreamer_customfilter_passthrough.so', 'custom']]
-extensions += [['custom', 'custom', nnstreamer_unittest_deps, '../build/nnstreamer_example/custom_example_passthrough/libnnstreamer_customfilter_passthrough_variable.so', 'custom-set']]
-
-if get_option('enable-tensorflow-lite')
-  extensions += [['tensorflow-lite', 'tflite', nnstreamer_filter_tflite_deps, 'mobilenet_v1_1.0_224_quant.tflite', 'tensorflow_lite']]
-  extensions += [['tensorflow-lite', 'tflite', nnstreamer_filter_tflite_deps, 'add.tflite', 'tensorflow_lite-set']]
-endif
-
-if get_option('enable-tensorflow')
-  extensions += [['tensorflow', 'tf', nnstreamer_filter_tf_deps, 'mnist.pb', 'tensorflow']]
-endif
-
-if get_option('enable-nnfw-runtime')
-  extensions += [['nnfw', 'nnfw', nnstreamer_filter_nnfw_deps, 'add.tflite', 'nnfw']]
-endif
-
-if get_option('enable-pytorch')
-  extensions += [['pytorch', 'torch', nnstreamer_filter_torch_deps, 'pytorch_lenet5.pt', 'pytorch']]
-endif
-
-if get_option('enable-caffe2')
-  extensions += [['caffe2', 'caffe2', nnstreamer_filter_caffe2_deps, 'caffe2_init_net.pb,caffe2_predict_net.pb', 'caffe2']]
-endif
-
-if have_python2
-  extensions += [['python2', 'python2', nnstreamer_filter_python2_deps, 'passthrough.py', 'python2-get']]
-  extensions += [['python2', 'python2', nnstreamer_filter_python2_deps, 'scaler.py', 'python2-set']]
-endif
-
-if have_python3
-  extensions += [['python3', 'python3', nnstreamer_filter_python3_deps, 'passthrough.py', 'python3-get']]
-  extensions += [['python3', 'python3', nnstreamer_filter_python3_deps, 'scaler.py', 'python3-set']]
-endif
+# {execute_when, name, ext_abbr, test_name, deps, model file/folder, env}
+
+py_env = environment()
+
+py_env.append('PYTHONPATH', join_paths(meson.build_root(), 'ext', 'nnstreamer', 'tensor_filter'))
+
+extensions = [
+  {
+    'execute_when': true,
+    'name': 'custom',
+    'abbr': 'custom',
+    'test_name': 'custom',
+    'deps': 'nnstreamer_unittest_deps',
+    'model': '../build/nnstreamer_example/custom_example_passthrough/libnnstreamer_customfilter_passthrough.so'
```

Comment:
Do not assume that the build root is 'build'. It is changable.

Reply:
Thanks for pointing out :)